cfgov/cfgov/settings/base.py

```diff
@@ -152,6 +152,11 @@
 
 ALLOWED_HOSTS = ['*']
 
+#Enabling compression for use in base.html
+COMPRESS_ENABLED = True
```

Comment:
If we're enabling compression in base, maybe we need to disable in local.py? I imagine it makes Javascript debugging harder (not an expert here, though)